migrations/versions/42cbae986650_policy_id_and_revoke.py

```diff
@@ -110,12 +118,15 @@ def downgrade():
             request_id, policy_id = r[0], r[1]
 
             if not config["LOCAL_MIGRATION"]:
-                resource_paths = get_resource_paths_for_policy(
-                    existing_policies["policies"], policy_id
-                )
-                assert (
-                    len(resource_paths) > 0
-                ), f"No resource_paths for policy {policy_id}"
+                resource_paths = policy_resource.get(policy_id)
+                if not resource_paths:
+                    resource_paths = get_resource_paths_for_policy(
```

Comment:
this is fine if you want to keep it, but it's not really necessary, as `get_resource_paths_for_policy` does not make any additional calls to Arborist